hack/lib/vars.bash

```diff
@@ -48,6 +48,7 @@ readonly UPGRADE_CLUSTER="${UPGRADE_CLUSTER:-"false"}"
 readonly UPGRADE_OCP_IMAGE="${UPGRADE_OCP_IMAGE:-}"
 
 readonly INSTALL_PREVIOUS_VERSION="${INSTALL_PREVIOUS_VERSION:-"false"}"
+readonly UNINSTALL_MESH="${UNINSTALL_MESH:-"false"}"
```

Comment:
This feels a little out of place. Should `mesh.sh` just take a parameter instead?

Reply:
done.